### PR TITLE
feat:[CDS-97321]: Clone inline environment and services

### PR DESCRIPTION
**Pull request by krishi0408**
Thanks for contributing to the Harness Developer Hub! Our code owners will review your submission.

## Description

* Please describe your changes: \__________________________________
* Jira/GitHub Issue numbers (if any): \______________________________
* Preview links/images (Internal contributors only): \__________________

## PR lifecycle

We aim to merge PRs within one week or less, but delays happen sometimes.

If your PR is open longer than two weeks without any human activity, please tag a [code owner](https://github.com/harness/developer-hub/blob/main/.github/CODEOWNERS) in a comment.

PRs must meet these requirements to be merged:

- [ ] Successful preview build.
- [ ] Code owner review.
- [ ] No merge conflicts.
- [ ] Release notes/new features docs: Feature/version released to at least one prod environment.

### CODE REVIEW

File: docs/continuous-delivery/x-platform-cd-features/environments/create-environments.md [hunk 1]
```diff
@@ -463,6 +463,26 @@ GitOps clusters are used in a PR pipeline. A PR pipeline creates and merges a Gi
 GitOps Clusters are not used in standard CD pipelines. They're used when using GitOps only.
 
 
+## Clone Environments
+
+:::info note
+Currently, this feature is behind the feature flag `CDS_SERVICE_ENV_CLONING`. Contact [Harness Support](mailto:support@harness.io) to enable the feature.
+This is currently being supported for only for Environments which is stored in Harness i.e INLINE entity.
```
Comment (SushrutHarness) [inline docs/continuous-delivery/x-platform-cd-features/environments/create-environments.md:470]:
```suggestion
Currently, this is only being supported for Environments that are stored in Harness (i.e INLINE entities).
```

File: docs/continuous-delivery/x-platform-cd-features/environments/create-environments.md [hunk 2]
```diff
@@ -463,6 +463,26 @@ GitOps clusters are used in a PR pipeline. A PR pipeline creates and merges a Gi
...
+:::
+
+You can clone environment across scopes i.e from one project to another, project to organization, account to project etc.
```
Comment (SushrutHarness) [inline docs/continuous-delivery/x-platform-cd-features/environments/create-environments.md:473]:
```suggestion
You can clone environment across scopes (i.e from one project to another, project to organization, account to project etc.).
```

File: docs/continuous-delivery/x-platform-cd-features/environments/create-environments.md [hunk 3]
```diff
@@ -463,6 +463,26 @@ GitOps clusters are used in a PR pipeline. A PR pipeline creates and merges a Gi
...
+
+Select **More Options** (&vellip;).
```
Comment (SushrutHarness) [inline docs/continuous-delivery/x-platform-cd-features/environments/create-environments.md:475]:
```suggestion
Select **More Options**.
```

File: docs/continuous-delivery/x-platform-cd-features/environments/create-environments.md [hunk 4]
```diff
@@ -463,6 +463,26 @@ GitOps clusters are used in a PR pipeline. A PR pipeline creates and merges a Gi
...
+Select **Clone**
+![](./static/clone_option_env_inline.png)
+
+Once you click on **Clone**, you will see **Clone Environment** setting:-
```
Comment (SushrutHarness) [inline docs/continuous-delivery/x-platform-cd-features/environments/create-environments.md:479]:
```suggestion
Once you click on **Clone**, you will see the **Clone Environment** setting:
```

File: docs/continuous-delivery/x-platform-cd-features/environments/create-environments.md [hunk 5]
```diff
@@ -463,6 +463,26 @@ GitOps clusters are used in a PR pipeline. A PR pipeline creates and merges a Gi
...
+![](./static/clone_env_inline_setting.png)
+
+You can change the **Name** add tags and description as well for this clone environment.
```
Comment (SushrutHarness) [inline docs/continuous-delivery/x-platform-cd-features/environments/create-environments.md:482]:
```suggestion
You can change the **Name**, add tags, or description for this clone environment.
```

File: docs/continuous-delivery/x-platform-cd-features/environments/create-environments.md [hunk 6]
```diff
@@ -463,6 +463,26 @@ GitOps clusters are used in a PR pipeline. A PR pipeline creates and merges a Gi
...
+
+You can change the **Organization** and **Project** as well where you want to clone the environment.
```
Comment (SushrutHarness) [inline docs/continuous-delivery/x-platform-cd-features/environments/create-environments.md:484]:
```suggestion
You can modify the destination of your clone environment using the  **Organization** and **Project** fields.
```

**Review by SushrutHarness — COMMENTED** [review]
left some Sugs

File: docs/continuous-delivery/x-platform-cd-features/services/services-overview.md [hunk 7]
```diff
@@ -20,6 +20,28 @@ When you create a service in a pipeline, it's automatically added to **Services*
 
 For more information, go to [create services](/docs/continuous-delivery/x-platform-cd-features/services/create-services).
 
+## Clone Services
+
+:::info note
+Currently, this feature is behind the feature flag `CDS_SERVICE_ENV_CLONING`. Contact [Harness Support](mailto:support@harness.io) to enable the feature.
+This is currently being supported for only for Environments which is stored in Harness i.e INLINE entity.
```
Comment (shylajasun) [inline docs/continuous-delivery/x-platform-cd-features/services/services-overview.md:27]: @krishi0408 in Services , there is Environment cloning. Could you fix this

**Review by SushrutHarness — APPROVED** [review]
After you fix Shylaja's comment, LGTM

**Review by shylajasun — APPROVED** [review]

**Pull request merged**